src/routes/AuthLayoutRouter/components/Drawer/Drawer.js

```diff
@@ -120,13 +132,17 @@ export default function Drawer (props) {
   )
 }
 
-export function ContextRow ({ currentLocation, group, routeParams, explicitPath }) {
+export function ContextRow ({
+  currentLocation,
+  group,
+  explicitPath
+}) {
   const { avatarUrl, context, name, newPostCount, slug } = group
   const imageStyle = bgImageStyle(avatarUrl || DEFAULT_AVATAR)
   const showBadge = newPostCount > 0
   const path = explicitPath || baseUrl({ context, groupSlug: slug })
   return (
-    <li styleName={cx('s.contextRow', { 's.currentContext': currentLocation?.pathname === path })}>
+    <li styleName={cx('s.contextRow', { 's.currentContext': currentLocation?.pathname === path || (path.includes(myPath) && currentLocation?.pathname.includes(myPath)) })}>
```

Comment:
why doesnt the existing match work for /my?

Reply:
because the prior matches were exact matches, not partial.

We want `/my/` to match four different URLs, where as the other contextRows have to match specific URLs one-to-one

![image](https://user-images.githubusercontent.com/5193218/210663417-fa607366-b40d-43a2-8864-de05cc7a4570.png)

Reply:
gotcha